docs/source/models/intervention_models/intrapartum/acs_intervention.rst

```diff
@@ -0,0 +1,234 @@
+  * - All (Ethiopia, Nigeria, Pakistan)
+    - CEmONC Facilities
+    - 76.29
+    - :math:`\text{Uniform}(66.63,89.23)`
+    - From Health Systems team estimates saved at ``/snfs1/Project/simulation_science/mnch_grant/MNCNH portfolio/sim_science_estimates.csv``
+
+
+Vivarium Modeling Strategy
+--------------------------
+
+Intervention eligibility criteria (see :ref:`the intrapartum intervention module document <2024_vivarium_mncnh_portfolio_intrapartum_interventions_module>` 
+for how to obtain this information in the MNCNH portfolio simulation):
+
+  1. Preterm birth is expected 
+  2. Delivers in facility with CPAP availability
+
+This intervention requires adding an attribute to all simulants who expect to give birth to a preterm infant (i.e., based on believed gestational age 34 to 36 weeks from pregnancy module output) and who give birth 
+in a facility with CPAP availability (based on ``cpap_availability``; see :ref:`the CPAP intervention document <intervention_neonatal_cpap>`) to specify if a parent-child dyad
+receives ACS or not.  We will track this and the model will have different mortality rates for preterm with RDS for parent-child dyads with and without 
+ACS (implemented with a slightly confusing application of our ``Risk`` and ``RiskEffect`` components from ``vivarium_public_health``).
+
+The ``Risk`` component adds an attribute to each simulant indicating whether the simulant has received ACS during the intrapartum period. Only simulants who expect to 
+give birth to a preterm infant and who give birth in a facility with CPAP availability are eligible for this intervention.
+
+To make this work naturally with the ``RiskEffect`` component, it is best to think of the risk as "no ACS".  With this framing, the ``RiskEffect`` 
+component requires data on (1) the relative risk of preterm with RDS mortality for people who did not receive ACS, and (2) the population attributable fraction (PAF) of preterm with
+RDS deaths due to not receiving ACS.  We will use the decision tree below to estimate the probability of preterm with RDS mortality with and without the use of ACS, ensuring consistency
+with the baseline delivery facility rates and baseline ACS coverage.
+
+In Vivarium, this risk effect will modify the preterm with RDS mortality pipeline, resulting in 
+
+.. math::
+
+   \text{CSMRisk}_i^\text{preterm with RDS} = \text{CSMRisk}^\text{preterm with RDS} \cdot (1 - \text{PAF}_\text{no ACS}) \cdot \text{RR}_i^\text{no ACS}
+
+where :math:`\text{RR}_i^\text{no ACS}` is simulant *i*'s individual relative risk for "no ACS", meaning :math:`\text{RR}_i^\text{no ACS} = \text{RR}_\text{no ACS}` 
+if simulant *i* does not receive ACS, and :math:`\text{RR}_i^\text{no ACS} = 1` if simulant *i* receives ACS. 
+
+The relative risk value we will use is pulled from [Gallos-et-al-2018-Cochrane-Review]_, the most recent Cochrane Review of the effect of 
+sublingually received ACS during labor on the prevention of preterm with RDS.
```

Comment:
Above you said the recommended drugs are delivered intramuscually.  It is not something that shows up in our sim, but it is the kind of thing that might make a reader wonder what else we have done inconsistently.  Please double check the mode of delivery and make sure it is consistent in our docs.